README.md

```diff
@@ -78,7 +78,8 @@ or the configuration file can be specified by cli args as well.
 * `make run-mock-registry`: Mock registry. Entirely separate binary.
 * `make test`: Runs the test suite.
 * `make vendor`: Updates the dependencies
-* `make build`: Builds a docker container of the current source
+* `make dev-image`: Builds a docker container of the current source
+* `make release-image` Builds a docker container using the latest rpm from [Copr](https://copr.fedorainfracloud.org/coprs/g/ansible-service-broker/ansible-service-broker/)
```

Comment:
@rthallisey AH, that's where you get `build` from the perspective of `docker build`. I always thought of it as compiling a binary which is why it felt weird to run `make build` to build a container.